Azure/azure-key-vault/ingest/parser.yml

```diff
@@ -0,0 +1,33 @@
+name: azure-key-vault
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
```

Comment:
We start withdrawing `event.kind: "event"` as most events in the platform are `event` (But we continue to keep this field for alerts and other kind).
So, In this case, I suggest removing this field:
```suggestion
```